lib/tasks/install_utils.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require_relative 'build_utils'
+require 'erb'
+require 'dotenv'
+require 'tempfile'
+
+module InstallUtils
+  include BuildUtils
```

Comment:
I think just 1 file is fine. It seems commonplace to have very large files in ruby right?

Name could change though. Happy to throw in helper or rake like `ood_rake_helper` or `ood_utilities`? :man_shrugging:

Reply:
lol we already have `rake_helper.rb`.

Reply:
There is already a `rake_helper` file, could just put everything in there?

Reply:
Yep, your second comment didn't show up right away for me, I'll put into `rake_helper.rb`.

Reply:
This change was made, moved all helper methods into `rake_helper.rb`